app/server/lib/NSandbox.ts

```diff
@@ -382,16 +382,16 @@ function isFlavor(flavor: string): flavor is keyof typeof spawners {
  * grist-core), and trying to regularize creation options a bit.
  *
  * The flavor of sandbox to use can be overridden by some environment variables:
- *   - GRIST_SANDBOX_FLAVOR: should be one of the spawners (pynbox, unsandboxed, docker,
- *     gvisor, macSandboxExec)
+ *   - GRIST_SANDBOX_FLAVOR: should be one of the spawners (gvisor, unsandboxed, docker,
+ *     macSandboxExec, pynbox)
  *   - GRIST_SANDBOX: a program or image name to run as the sandbox.  Not needed for
```

Comment:
I think this comment (about `GRIST_SANDBOX`) is a bit misleading too now. In particular, I think it's optional for gvisor and unsandboxed (but makes it sound like it's required except for pynbox).

Reply:
You're right, and there's other stale information in this file. If you don't mind I'm not going to pull any further at this thread right now.